tests/ui/cast/ptr-to-trait-obj-add-auto.rs

```diff
@@ -1,18 +1,20 @@
-//@ check-pass
-
 trait Trait<'a> {}
 
 fn add_auto<'a>(x: *mut dyn Trait<'a>) -> *mut (dyn Trait<'a> + Send) {
     x as _
-    //~^ warning: adding an auto trait `Send` to a trait object in a pointer cast may cause UB later on
-    //~| warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
+    //~^ ERROR cannot add auto trait `Send` to dyn bound via pointer cast
+    //~| NOTE unsupported cast
+    //~| NOTE this could allow UB elsewhere
+    //~| HELP use `transmute` if you're sure this is sound
```

Comment:
(curiosity) Any specific reason you are using `ERROR` instead of `error:`?

I prefer the latter cause it's closer to the actual compiler output (`error[CODE]: message` or `error: message` for code-less errors) and because in my opinion it looks/reads nicer. (but there is no accepted style, so feel free to use whatever of course)

Reply:
No particular reason.  `ERROR` just seemed more common among tests when I picked up the habit.